<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo/icon.png</normaloff>logo/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="planning_tab">
       <attribute name="title">
        <string>Liste</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="full_list_tab_layout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tools_tab">
       <attribute name="title">
        <string>Outils</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="tools_tab_layout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="about_menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="about_qt_action"/>
   </widget>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="load_profile_action"/>
    <addaction name="save_profile_action"/>
    <addaction name="close_profile_action"/>
    <addaction name="separator"/>
    <addaction name="quit_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="about_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="quit_action">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/delete.ico</normaloff>:/icons/icons/delete.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="about_qt_action">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/delete.ico</normaloff>:/icons/icons/delete.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;A propos de Qt</string>
   </property>
  </action>
  <action name="load_profile_action">
   <property name="text">
    <string>Charger un profil</string>
   </property>
  </action>
  <action name="close_profile_action">
   <property name="text">
    <string>Fermer le profil</string>
   </property>
  </action>
  <action name="save_profile_action">
   <property name="text">
    <string>Sauvegarder le profil</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>A propos ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
